Tangem/Modules/ManageTokens/ManageTokensViewModel.swift

```diff
@@ -21,21 +21,36 @@ final class ManageTokensViewModel: ObservableObject {
 
     @Published var tokenViewModels: [ManageTokensItemViewModel] = []
     @Published var isLoading: Bool = true
+    @Published var hasPendingDerivations: Bool = false
```

Comment:
так и получается что здесь можно 
```suggestion
    @Published var generateAddressesViewModel: GenerateAddressesViewModel?
```

будет круче )

Reply:
Ага поправил